pkg/bpf/prog.go

```diff
@@ -67,3 +74,72 @@ func (t ProgType) String() string {
 
 	return "Unknown"
 }
+
+type attrProg struct {
+	startID   uint32
+	progID    uint32
+	nextID    uint32
+	openFlags uint32
+}
+
+// ProgInfo holds values from the upstram struct bpf_prog_info.
+// From: https://github.com/torvalds/linux/blob/master/include/uapi/linux/bpf.h#L1006
```

Comment:
Might be good to refer to a permalink, such as https://github.com/torvalds/linux/blob/v4.16/include/uapi/linux/bpf.h#L924 .

Reply:
added :)